.commitlintrc.yml

```diff
@@ -0,0 +1,82 @@
+# Commitlint configuration.
+# See: https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md
+---
+rules:
+  body-leading-blank:
+    - 2
+    - always
+  body-max-line-length:
+    - 2
+    - always
+    - 75
+  footer-leading-blank:
+    - 2
+    - always
+  header-max-length:
+    - 2
+    - always
+    - 75
+  scope-enum:
+    - 2
+    - always
+    - - advisor
+      - analyzer
+      - cli
+      - clients
+      - deps
+      - detekt
+      - docker
+      - downloader
+      - evaluator
+      - examples
+      - github
+      - helper
+      - integrations
+      - model
+      - notifier
+      - reporter
+      - scanner
+      - utils
+  scope-case:
+    - 2
+    - always
+    - lower-case
+  subject-case:
+    - 2
+    - always
+    - - pascal-case
+      - sentence-case
+      - start-case
+      - upper-case
+  subject-empty:
+    - 2
+    - never
+  subject-full-stop:
+    - 2
+    - never
+    - .
+  type-case:
+    - 2
+    - always
+    - kebab-case
```

Comment:
Why not lower-case like for `scope`?

Reply:
Copy & paste from your config 😉

Reply:
Actually, I believe `lower-case` *only* is enough, as we also configure `type-enum`.